### ISSUE TITLE
Which dataset of C2C12 was used in your paper?

### ISSUE THREAD

**Issue by imzhangyd** [talbenha/cell-tracker-gnn#11]
Hi, thanks for your wonderful work! I want to download the C2C12 dataset, but I found there are 3 folders and each includes 16 sequences, see the figure below. I want to learn which was used in your paper?
![image](https://github.com/talbenha/cell-tracker-gnn/assets/39505533/c0f51b79-583d-49e4-80c7-d6cadc49cfd9)

**Comment by talbenha** [comment]
As mentioned in the paper, we followed the partition used in previous works. 

Here is a summary of our correspondence with Kazuya Nishimura (email address: [kazuya.nishimura@human.ait.kyushu-u.ac.jp](mailto:kazuya.nishimura@human.ait.kyushu-u.ac.jp)):
1. Training data:
We used 4 sequences.
 1. Sequence 9 of Dataset01 (3rd March 2009). We used 100 images from 680 to 780. 
 2.  Sequence 2 of Dataset01 (3rd March 2009). We used 100 images from 600 to 700.
 3. Sequence 17 of Dataset 02 (18th March 2009). We used 100 images from 600 to 700.
 4. Sequence 18 of Dataset 02 (18th March 2009). We used 100 images from 600 to 700.
The annotation of sequence 9 is publicly available from this website https://www.nature.com/articles/sdata2018237.pdf .
The other 3 sequences have been uploaded to github https://github.com/JunyaHayashida/MPM/tree/master/additional_annotation
2. Test data:
![image](https://github.com/talbenha/cell-tracker-gnn/assets/57532696/28f93933-04ab-4a4d-af9e-22892ca5c780)
